packages/playground/src/types/index.ts

```diff
@@ -63,6 +63,7 @@ export interface K8SWorker {
   rootFsSize: number;
   rentedBy?: number;
   dedicated: boolean;
+  rentedByMe?: boolean;
```

Comment:
do we still need rentedByMe?

Reply:
same usage https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3852#discussion_r1963724208